src/component/strategies/StrategiesList/StrategiesList.tsx

```diff
@@ -265,81 +269,105 @@ export const StrategiesList = () => {
-            }
-            elseShow={
-                <Tooltip title="You cannot edit a built-in strategy" arrow>
-                    <div>
-                        <IconButton disabled size="large">
-                            <Edit titleAccess="Edit strategy" />
-                        </IconButton>
-                    </div>
-                </Tooltip>
-            }
-        />
-    );
+                }
+                elseShow={
+                    <Tooltip title="You cannot edit a built-in strategy" arrow>
+                        <div id={id}>
+                            <IconButton
+                                disabled
+                                size="large"
+                                aria-labelledby={id}
+                            >
+                                <Edit titleAccess="Edit strategy" />
+                            </IconButton>
+                        </div>
+                    </Tooltip>
+                }
+            />
+        );
+    };
 
-    const deleteButton = (strategy: IStrategy) => (
-        <ConditionallyRender
-            condition={strategy?.editable}
-            show={
-                <PermissionIconButton
-                    onClick={() => onDeleteStrategy(strategy)}
-                    permission={DELETE_STRATEGY}
-                    tooltipProps={{ title: 'Delete strategy' }}
-                >
-                    <Delete />
-                </PermissionIconButton>
-            }
-            elseShow={
-                <Tooltip title="You cannot delete a built-in strategy" arrow>
-                    <div>
-                        <IconButton disabled size="large">
-                            <Delete titleAccess="Delete strategy" />
-                        </IconButton>
-                    </div>
-                </Tooltip>
-            }
-        />
-    );
+    const deleteButton = (strategy: IStrategy) => {
+        const id = createStrategyId(strategy, 'delete');
+        return (
+            <ConditionallyRender
+                condition={strategy?.editable}
+                show={
+                    <PermissionIconButton
+                        onClick={() => onDeleteStrategy(strategy)}
+                        permission={DELETE_STRATEGY}
+                        tooltipProps={{ title: 'Delete strategy' }}
+                    >
+                        <Delete />
+                    </PermissionIconButton>
+                }
+                elseShow={
+                    <Tooltip
+                        title="You cannot delete a built-in strategy"
+                        arrow
+                    >
+                        <div id={id}>
+                            <IconButton
+                                disabled
+                                size="large"
+                                aria-labelledby={id}
+                            >
+                                <Delete titleAccess="Delete strategy" />
```

Comment:
In cases where it's disabled, do we announce that to assistive tech? Should the `titleAccess` instead be "you cannot delete a built-in strategy" or is that announced in some other way?

Reply:
This change seems to have disappeared (I'm guessing you removed it), but I'm still curious about how this acts. Do you know?

Reply:
Yeah, there were extensive changes to these components in main, also partly focused on a11y.

Reply:
Combined the labels into one for the disabled buttons. Should be better, though probably still improvable.